### PR TITLE
Added Altair GraphQL Client usage to docs

### PR DESCRIPTION
**Pull request by imolorhe**
Added [Altair GraphQL Client](https://altair.sirmuel.design/) usage to the docs.

### CODE REVIEW

**Comment by imolorhe** [comment]
@sungam3r @SimonCropp Please review.

File: docs2/site/docs/getting-started/altair-graphql.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+The easiest way to add Altair into your ASP.NET Core app is to use the [GraphQL.Server.Ui.Altair](https://www.nuget.org/packages/GraphQL.Server.Ui.Altair) package.
+All you need to do after installing nuget is to append one extra line in your `Startup.cs`:
+```c#
+public void Configure(IApplicationBuilder app, IHostingEnvironment env)
+{
+    app.UseGraphQLAltair();
+}
+```
+If you do not explicitly specify an endpoints through the optional `options` argument then
+GraphiQL by default will run on `/altair` endpoint and will send requests to `/graphql`
```
Comment (sungam3r) [inline docs2/site/docs/getting-started/altair-graphql.md]:
```suggestion
Altair by default will run on `/altair` endpoint and will send requests to `/graphql`
```

Comment (sungam3r) [reply docs2/site/docs/getting-started/altair-graphql.md]: I don't have permissions to push into your repo, so you have to fix this yourself.

Comment (imolorhe) [reply docs2/site/docs/getting-started/altair-graphql.md]: Applied the suggestion 🙂

**Pull request merged**